### PR TITLE
Update User Guide - Help, Create Module and Class

### LINKED ISSUE #50: Update User Guide: Viewing Help, Create module, Create class

### PR DESCRIPTION
**Pull request by caipng**
Closes #50

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
 
    * **`list`** : Lists all contacts.
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * **`add`**`n/John Doe p/98765432 e/johnd@example.com` : Adds a contact named `John Doe` to the Address Book.
```
Comment (chaiwanlin) [inline docs/UserGuide.md:29]:
```suggestion
   * **`add`**`n/John Doe p/98765432 e/johnd@example.com` : Adds a contact named `John Doe` to EdRecord.
```

**Review by chaiwanlin — CHANGES_REQUESTED** [review]
Just a small change, otherwise LGTM

File: docs/UserGuide.md (hunk 1 above)
Comment (caipng) [reply docs/UserGuide.md]: Good catch

**Review by chaiwanlin — APPROVED** [review]
LGTM

**Pull request merged**